workloads/librispeech/librispeech_jax/input_pipeline.py

```diff
@@ -0,0 +1,55 @@
+"""Data pipeline for LibriSpeech dataset modified from https://github.com/lsari/librispeech_100/blob/main/dataset.py."""
+
+import json
+
+import jax.numpy as jnp
+import numpy as np
+import pandas as pd
+import torch
+
+
+class LibriSpeechDataset(torch.utils.data.Dataset):
```

Comment:
now that we are requiring pytorch be installed for jax workloads (which is fine), we should change `setup.py` to always install pytorch